src/workflomics_benchmarker/cwltool_wrapper.py

```diff
@@ -47,24 +52,44 @@ def check_cwltool(self):
             print("cwltool is not installed.")
         return version
 
-    def update_input_yaml(self, input_yaml_path):
-        """Update the input yaml file with the paths to the input files"""
+    def update_input_yaml(self, input_yaml_path:str, interactive: bool) -> dict:
+        """
+        Update the input yaml file with the paths to the input files. 
+        If interactive is set to True, the user will be allowed to edit the paths to the input files.
+        
+        Parameters
+        ----------
+        input_yaml_path : str
+            The path to the input yaml file.
+            
+        interactive : bool
+            If True, the user will be allowed to edit the paths to the input files.
+        
+        Returns
+        -------
+        dict
+            A dictionary containing the input file names.
+        """
         inputs = {}
         with open(input_yaml_path, 'r') as file:
             input_data = yaml.safe_load(file)
 
         for key, value in input_data.items():
             if key.startswith('input'):
-                print(f"The path for {key} is {value['path']}. Do you want to change it? (y/n)")
+                if not interactive:
+                    inputs[key] = {"filename": Path(value['path']).name}
+                    continue
+
+                print(f"The path for {key} is {value['path']}. Would you like to keep this path? ([Y]/n)")
                 answer = input()
```

Comment:
Maybe convert to lowercase to also accept Y and N?

Reply:
I agree, I fixed it! I also included `no` as a valid option.